zipline/finance/blotter/blotter_utils.py

```diff
@@ -0,0 +1,112 @@
+from functools import partial
```

Comment:
Could we call this something like `dispatcher.py` so it is more clear what this contains?